File: app/src/main/java/com/qiuchenly/comicx/ViewCreator/RectCardViewDefault.java
package com.qiuchenly.comicx.ViewCreator;


import android.content.Context;
import android.util.AttributeSet;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

class RectCardViewDefault extends RectCardView {

    public RectCardViewDefault(@NonNull Context context) {
        super(context);
    }

    public RectCardViewDefault(@NonNull Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    public RectCardViewDefault(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    int getViewHeight(int widthMeasureSpec, int heightMeasureSpec) {
        return widthMeasureSpec;
    }
}

File: app/src/main/java/com/qiuchenly/comicx/ViewCreator/RectCardViewDefault.kt
package com.qiuchenly.comicx.ViewCreator


import android.content.Context
import android.util.AttributeSet

internal class RectCardViewDefault : RectCardView {

    constructor(context: Context) : super(context)

    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs)

    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
        context,
        attrs,
        defStyleAttr
    )

    internal override fun getViewHeight(widthMeasureSpec: Int, heightMeasureSpec: Int): Int {
        return widthMeasureSpec
    }
}